src/main/java/fm/last/moji/impl/MojiFileAttributesImpl.java

```diff
@@ -65,6 +77,12 @@ public String toString() {
     builder.append(length);
     builder.append(", fid=");
     builder.append(fid);
+
+    if (checksum != null) {
```

Comment:
We don't need this check either, I think it's fine to output "checksum=null" when it is null.